package/batocera/core/batocera-system/batocera.conf

```diff
@@ -248,8 +255,22 @@ global.retroachievements.password=
 #snes.retroarch.menu_driver=rgui
 #global.retroarch.input_max_users=4
 
+## Wiimotes
+## Real wiimotes must not be paired with batocera.linux system so that they can work with the Wii emulator
+## Set emulatedwiimotes to 1 to emulate wiimotes with standard pads
+#wii.emulatedwiimotes=0
+
+## PS2
+# Enables fast booting (skip PS2 BIOS when booting an ISO ; some games relies on BIOS to get start-up parameters and thus may not work correctly)
+# Set fullboot to 0 to disable BIOS when booting an ISO
+#ps2.fullboot=1
+
 # Scraper
 # Comma seperated order to prefer images, s=snapshot, b=boxart, f=fanart, a=banner, l=logo, 3b=3D boxart
 #scrapper.style=s,b,f,a,l,3b
 
+# Enable DXVK for Wine and the HUD for see FPS.
```

Comment:
```
## Enable DXVK for WINE and the HUD option to enable FPS counter --- x86 series specific
```

Does it matter if we disable the options?
In fact
`#windows64.dxvk=0` should equal `windows64.dxvk=0`

Reply:
Copied all option without paying attention. But yes, should be the same.